geode-core/src/main/java/org/apache/geode/internal/cache/PRHARedundancyProvider.java

```diff
@@ -1711,9 +1712,12 @@ public boolean recoverPersistentBuckets() {
     /*
      * Start the redundancy logger before recovering any proxy buckets.
      */
-    startRedundancyLogger(proxyBucketArray.length);
+    persistentBucketRecoverer = createPersistentBucketRecoverer(proxyBucketArray.length);
+    Thread loggingThread = new LoggingThread(
```

Comment:
I think this loggingThread should be part of the PersistentBucektRecoverer class. I see no need for this provider class to know about the loggingThread.